README.md

```diff
@@ -1,13 +1,65 @@
-# Temporian
+![Temporian logo](resources/banner.png)
 
-**Temporian** is a library to pre-process temporal
-signals before their use as input features with off-the-shelf tabular machine
-learning libraries (e.g., TensorFlow Decision Forests).
+![tests](https://github.com/google/temporian/actions/workflows/test.yaml/badge.svg) ![formatting](https://github.com/google/temporian/actions/workflows/formatting.yaml/badge.svg)
 
-![tests](https://github.com/google/temporian/actions/workflows/test.yaml/badge.svg)
-![formatting](https://github.com/google/temporian/actions/workflows/formatting.yaml/badge.svg)
 
-## Requirements
+**Temporian** is a library to pre-process temporal signals before their use as input features with off-the-shelf tabular machine learning libraries (e.g., TensorFlow Decision Forests, scikit learn, XGBoost).
+
+
+## Usage Example
+
+A minimal end-to-end run looks as follows:
+
+```python
+import temporian as t
+
+# Load the data.
+event_data = t.read_event("path/to/data.csv")
+
+# Event schema
+event = t.Event([t.Feature("sales", int), t.Feature("costs", int)])
+
+# Create Simple Moving Average feature
+sma = t.simple_moving_average(
+    input=event,
+    window_length=t.day(5),
+)
+
+# Create Lag feature
+lag = t.lag(
+    input=event,
+    lag=t.week(1),
+)
+
+# Assign features
+output_event = t.assign(event, sma)
+output_event = t.assign(output_event, lag)
+
+
+# Execute pre processing functions and get results
+output_event = t.evaluate(
+    output_event,
+    input_data={
+        event: event_data,
+    },
+)
+
+```
+
+>__Warning__: The library is still under construction. This example usage is what we are aiming to build in the short term.
+
+## Supported Features
+Temporian currently supports the following features for pre-processing your temporal data:
```

Comment:
Some formatting of the markdown file seems off, `prettier` or a similar vscode extension should fix that for you